src/Microsoft.IIS.WebManager/app/main/app.component.ts

```diff
@@ -67,36 +68,40 @@ import { ServerAnalyticService } from '../webserver/server-analytic.service';
 })
 export class AppComponent implements OnInit, OnDestroy {
     constructor(private _router: Router,
-                private _connectService: ConnectService,
-                private _loadingSvc: LoadingService,
-                private _windowService: WindowService,
-                private _versionService: VersionService,
-                private _serverAnalyticService: ServerAnalyticService,
-                private _renderer: Renderer,
-                angulartics2: Angulartics2,
-                angulartics2GoogleAnalytics: Angulartics2GoogleAnalytics) {
+        private _connectService: ConnectService,
+        private _loadingSvc: LoadingService,
+        private _windowService: WindowService,
+        private _versionService: VersionService,
+        private _serverAnalyticService: ServerAnalyticService,
+        @Inject("Runtime") private runtime: Runtime,
```

Comment:
I take it this comes from WAC. They prescribe us to destroy it when our component is removed?

Reply:
This object is created by me. In the standard deployment scenario it is an empty object with a bunch of no-ops. In WAC scenario there are some operations that would take place including initialization their context and destroying